docs/examples/website-to-api/test_models.py

```
@@ -0,0 +1,67 @@
+
+# API base URL
+BASE_URL = "http://localhost:8000"
+
+def test_model_management():
+    """Test the model management endpoints."""
+    
+    print("=== Testing Model Management ===")
+    
+    # 1. List current models
+    print("\n1. Listing current models:")
+    response = requests.get(f"{BASE_URL}/models")
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)